brevbestilling-integrasjon/src/main/resources/api/brevbestilling-models.yaml

```diff
@@ -18,7 +18,7 @@ components:
         meldingId:
           type: string
         brevbestillingRequest:
-          $ref: "#/components/schemas/brevBestilling"
+          $ref: "#/components/schemas/brevBestillingRequest"
```

Comment:
Dette er feil. brevBestilling er lagd med intensjon å vare re-usable. Det er rikti at BrevBestillingRequest og brevBestilling er to forskellige ting.

Reply:
F.ex. BREV_ER_BESTILT event vil vare en komposisjon av BrevBestilling, BrevMetadata og Execution state.  Hvis BrevBestillingRequest ender opp på et queue den vil også ha execution state.

Reply:
Okay. Fjerner den endringen :)